src/Sylius/Bundle/OrderBundle/Resources/config/validation/Adjustment.xml

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<!--
-
- This file is part of the Sylius package.
-
- (c) Paweł Jędrzejewski
-
- For the full copyright and license information, please view the LICENSE
- file that was distributed with this source code.
-
--->
-
-<constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
-    <class name="Sylius\Component\Order\Model\Adjustment">
-        <property name="type">
-            <constraint name="NotBlank">
-                <option name="message">sylius.adjustment.type.not_blank</option>
-                <option name="groups">sylius</option>
-            </constraint>
-        </property>
-        <property name="amount">
-            <constraint name="NotBlank">
-                <option name="message">sylius.adjustment.amount.not_blank</option>
-                <option name="groups">sylius</option>
-            </constraint>
-            <constraint name="Type">
-                <option name="type">integer</option>
-                <option name="message">sylius.adjustment.amount.integer</option>
-                <option name="groups">sylius</option>
-            </constraint>
-        </property>
-    </class>
-</constraint-mapping>
```

Comment:
I guess that we don't need validation for an adjustment as much as we don't need form type for this entity. Should I revert this change?